app/components/provider_interface/interview_form_component.html.erb

```diff
@@ -16,7 +16,16 @@
     </div>
   <% end %>
 
-  <%= f.govuk_date_field :date, hint: { text: t('helpers.hint.provider_interface_interview_wizard.date', example_date: example_date) } %>
+  <%= f.govuk_date_field(
+        :date,
+        hint: {
+          text: t(
+            'helpers.hint.provider_interface_interview_wizard.date',
+            rbd_date: application_choice.reject_by_default_at.to_s(:govuk_date),
```

Comment:
Is it ever possible for `reject_by_default_at` to be `nil` here?

Reply:
Theoretically it can be nil. In practice, applications need to have been submitted before interviews can be arranged, so the rbd date should have been set